burn-tensor/src/tensor/ops/modules/pool.rs

```diff
@@ -16,7 +18,7 @@ pub(crate) fn avg_pool1d_from_2d<B: Backend>(
     B::reshape(x, Shape::from([batch_size, channels, length]))
 }
 
-pub(crate) fn avg_pool1d_backward_from_2d<B: Backend>(
+pub(crate) fn avg_pool1d_backward_from_avg_pool2d<B: Backend>(
```

Comment:
I prefer using `from_2d` alone, as `from_avg_pool2d` is technically incorrect since it uses `avg_pool2d_backward`. The extended name is redundant, therefore `from_2d` suffices.